### PR TITLE
OP-21844 Fixed postgresql cve by upgrading it to 4.7.2. CVE-2024-1597

### PR DESCRIPTION
**Pull request by sanopsmx**
Jira : https://devopsmx.atlassian.net/browse/OP-21844
Summary : Upgraded postgresql version
Testing :

compile successful, no new TCs failed bcoz of this.
service started successfully after this change.
Created trivy-scan locally, this CVE not found
Pre-merge : NA
Post-merge : QA regression testing

### CODE REVIEW

File: fiat-sql-postgres/fiat-sql-postgres.gradle [hunk 1]
```diff
@@ -1,3 +1,9 @@
 dependencies {
-    runtimeOnly "org.postgresql:postgresql"
+    runtimeOnly "org.postgresql:postgresql:42.7.2"
```
Comment (aman-agrawal) [inline fiat-sql-postgres/fiat-sql-postgres.gradle:2]: why explicitly mentioning the version ? applicable elsewhere

**Comment by aman-agrawal** [comment]
how we are sure that this change (version upgrade) will not break the functionality ?

**Comment by sanopsmx** [comment]
> how we are sure that this change (version upgrade) will not break the functionality ?

Service is coming up...QA needs to test as discussed with Jay.

File: fiat-sql-postgres/fiat-sql-postgres.gradle (hunk 1 above)
Comment (sanopsmx) [reply fiat-sql-postgres/fiat-sql-postgres.gradle:2]: Because it is fetching version 42.5.4 from the spring boot dependencies pom.

Comment (aman-agrawal) [reply fiat-sql-postgres/fiat-sql-postgres.gradle:2]: [Suggestion] : Add dependency in kork-bom & fetch it from there.

**Comment by sanopsmx** [comment]
@aman-agrawal Closing this PR as there are no changes in fiat.

**Pull request closed without merging**